src/main/java/com/nvs/store/config/SecurityConfig.java

```diff
@@ -17,22 +17,35 @@ public class SecurityConfig {
     private final JwtAuthFilter jwtAuthFilter;
     private final AuthenticationProvider authenticationProvider;
 
+//    @Bean
```

Comment:
revert

Reply:
please remove security only in local development and testing - main functionality should have it

Reply:
done